pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -55,7 +53,6 @@ in
 , cowsay
 , hello
   # Overridable configurations
-, enableNvidiaContainerCli ? true
   # --nvccli currently requires extra privileges:
   # https://github.com/apptainer/apptainer/issues/1893#issuecomment-1881240800
 , forceNvcCli ? false
```

Comment:
THer'es a `nvidia-container-cli`-related option left, but we're deleting `libnvidia-container` which provides it? (I still suspect we shouldn't but I haven't looked into this in detail)